### PR TITLE
added keycloack config option

### PR DESCRIPTION
**Pull request by yashnalla**

### CODE REVIEW

**Comment by yashnalla** [comment]
There are issues with the logout routing see: https://github.com/RedHatInsights/insights-chrome/pull/321 that i was able to locally solve by substituting /logout.html for /logout

File: src/js/jwt/jwt.js [hunk 1]
```diff
@@ -219,7 +220,7 @@ function loginAllTabs() {
 /*** User Functions ***/
 // Get user information
 exports.getUserInfo = () => {
-    log('Getting User Information');
+    log('Getting User Information changes!!!');
```
Comment (iphands) [inline src/js/jwt/jwt.js]: Was the "changes!!!" string left in by mistake?

File: src/js/jwt/jwt.js [hunk 2]
```diff
@@ -237,6 +238,12 @@ exports.getUserInfo = () => {
     });
 };
 
+// Challenge auth and login if the user could be logged in, but in an unauth state
+exports.challengeAuth = () => {
```
Comment (iphands) [inline src/js/jwt/jwt.js]: I hate the name here. That is not a challenge. An auth challenge is whenever a user *must* re-enter their username/pasword.

Doesnt it make way more sense to just export this as login?

File: src/js/jwt/jwt.js (hunk 1 above)
Comment (yashnalla) [reply src/js/jwt/jwt.js]: Uhh im still working on this branch, so it was left in purposefully.

File: src/js/jwt/jwt.js (hunk 2 above)
Comment (yashnalla) [reply src/js/jwt/jwt.js]: Same as above. I was just gonna rename the function login

Comment (yashnalla) [reply src/js/jwt/jwt.js]: Well not login cause thats taken but something like checkAuth

**Pull request closed without merging**